crates/bins/wafl/src/commands/rpc/invoke.rs

```diff
@@ -53,7 +53,7 @@ pub(crate) async fn handle(opts: Options) -> Result<()> {
   let _guard = crate::utils::init_logger(&opts.logging)?;
 
   let mut client = wasmflow_rpc::make_rpc_client(
-    format!("http://{}:{}", opts.connection.address, opts.connection.port),
+    format!("http://{}:{}", opts.connection.host(), opts.connection.port()),
```

Comment:
My quick solution to allow URLs was to expose previously used `address` and `port` fields as methods.